MAINTAINERS_GUIDE.md

```diff
@@ -63,35 +66,54 @@ by anyone is denoted by adding a comment in the pull request: `LGTM`.
+of the project to maintain conceptual integrity.  Large decisions and 
+architecture changes should be reviewed by the chief maintainer.  
+The current chief maintainer for the project is Michael Crosby (@crosbymichael).  
 
-* Step 1: Determine the subdirectories affected by the pull request. This
-  might be `netlink/` and `security/`, or any other part of the repo.
+Even though the maintainer system is built on trust, if there is a conflict
+with the chief maintainer on a decision, their decision can be challenged 
+and brought to the technical oversight board if two third of the 
+maintainers vote for an appeal. It is expected that this would be a 
+very exceptional event.
 
-* Step 2: Find the `MAINTAINERS` file which affects this directory. If the
-  directory itself does not have a `MAINTAINERS` file, work your way up
-  the repo hierarchy until you find one.
 
-### I'm a maintainer, and I'm going on holiday
+### How are maintainers added?
 
-Please let your co-maintainers and other contributors know by raising a pull
-request that comments out your `MAINTAINERS` file entry using a `#`.
+The best maintainers have a vested interest in the project.  Maintainers
+are first and foremost contributors that have shown they are committed in
+the long term success of the project.  Contributors wanting to become 
+maintainers are expected to be deeply involved contributing code, 
+pull request review, and triage of issues in the project for more than two months.
 
-### I'm a maintainer, should I make pull requests too?
+Just contributing does not make you a maintainer, it is about building trust 
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.  The
+final vote to add a new maintainer should be approved by over 66% of the current
+maintainers with the chief maintainer having veto power.  In case of a veto, conflict resolution rules expressed above apply.  The voting period is
+five business days on the Pull Request to add the new maintainer.
 
-Yes. Nobody should ever push to master directly. All changes should be
-made through a pull request.
 
-### Who assigns maintainers?
+### What is expected of maintainers?
+
+Part of a healthy project is to have active maintainers to support the community
+in contributions and perform tasks to keep the project running.  Maintainers are
+expect to be able to respond in a timely manner if their help is required on specific
+issues where they are pinged.  Being a maintainer is a time consuming commitment and should
+not be taken lightly.
+
+When a maintainer is unable to perform the required duties they can be removed with
+a vote by 66% of the current maintainers with the chief maintainer having veto power.
+The voting period is ten business days.  Issues with a maintainer's performance should
+be discussed with them among the other maintainers so that they are not blind sided by
```

Comment:
I'd change “among the other maintainers” to “in advance”, since I don't think it matters whether the discussion is public or not and I had trouble parsing the original phrasing ;).